commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/configuration/VersionConfiguration.java

```diff
@@ -34,4 +34,7 @@
 	@Meta.AD(deflt = "true", name = "enabled", required = false)
 	public boolean enabled();
 
+	@Meta.AD(deflt = "5", name = "commerce-version-cap", required = false)
+	public int versionCap();
```

Comment:
Hey Alec, can you spell out "capacity" in both the field name and the language key?

Reply:
Got it. Is Capacity ok or would you rather it be Threshold?

Reply:
Ok.... new plan:
* Label: "Versions to Retain"
* Field name: versionsThreshold